moveit_core/CMakeLists.txt

```diff
@@ -158,35 +157,35 @@ message(STATUS " *** Building MoveIt ${MOVEIT_VERSION} ***")
 configure_file("version/version.h.in" "${VERSION_FILE_PATH}/moveit/version.h")
 install(FILES "${VERSION_FILE_PATH}/moveit/version.h" DESTINATION include/moveit)
 
-add_subdirectory(version)
-add_subdirectory(macros)
+add_subdirectory(background_processing)
 add_subdirectory(backtrace)
+add_subdirectory(collision_distance_field)
+add_subdirectory(constraint_samplers)
+add_subdirectory(controller_manager)
+add_subdirectory(distance_field)
+add_subdirectory(dynamics_solver)
 add_subdirectory(exceptions)
+add_subdirectory(kinematics_base)
+add_subdirectory(kinematic_constraints)
+add_subdirectory(macros)
```

Comment:
Alphabetized? macros should be below kinematic_metrics, collision_detection_* is still at the bottom

Reply:
If collision_detection_* is alphabetized, there is this compiler error:

CMake Error at collision_detection/CMakeLists.txt:42 (ament_add_gtest):
  Unknown CMake command "ament_add_gtest".

I don't know why but I don't intend to fix it in this PR. Will leave a TODO